src/main/java/mate/controller/AddCarController.java

```diff
@@ -0,0 +1,48 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+public class AddCarController extends HttpServlet {
+    private static final Injector injector
+            = Injector.getInstance("mate");
+    private final ManufacturerService manufacturerService
+            = (ManufacturerService) injector.getInstance(ManufacturerService.class);
+    private final CarService carService
+            = (CarService) injector.getInstance(CarService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        List<Manufacturer> manufacturers = manufacturerService.getAll();
+        request.setAttribute("manufacturers", manufacturers);
+        if (manufacturers.size() != 0) {
+            request.setAttribute("idMinValue", manufacturers.get(0).getId());
+            request.setAttribute("idMaxValue", manufacturers.get(manufacturers.size() - 1).getId());
+        }
```

Comment:
you don't need these values, you shouldn't show a user ids of manufacturers only their names, the same for other similar case